data/channels.csv

```diff
@@ -27653,8 +27654,10 @@ StarCinema.ru,Star Cinema,,,,RU,,,c/RU,rus,movies,FALSE,2018-08-06,,,http://star
 StarCinema.ua,Star Cinema,,,,UA,,,c/UA,ukr,movies,FALSE,2016-02-01,,,http://starcinema.starmediatv.ru/,https://i.imgur.com/5sis895.png
 StarCinema.us,Star Cinema,,,,US,,,c/US,eng,movies,FALSE,,,,,https://i.imgur.com/ObViBTR.png
 StarClassics.us,Star Classics,,,,US,,,c/US,eng,classic,FALSE,,,,,https://i.imgur.com/cv2XIQQ.png
+StarComedy.pt,Star Comedy,,The Walt Disney Company Portugal,The Walt Disney Company Iberia S.L.,PT,,Lisboa,c/PT,por,comedy,FALSE,2024-02-07,,,https://www.startv.pt/programacao/starcomedy,https://upload.wikimedia.org/wikipedia/commons/thumb/5/5a/Star_Comedy_2024.svg/1200px-Star_Comedy_2024.svg.png

```

Comment:
Image must be under 512px.

Reply:
Replace with https://upload.wikimedia.org/wikipedia/commons/thumb/5/5a/Star_Comedy_2024.svg/512px-Star_Comedy_2024.svg.png